src/pkg/securityhub/dao/security.go

```diff
@@ -131,3 +231,118 @@ func (d *dao) DangerousCVEs(ctx context.Context, scannerUUID string, projectID i
 	_, err = o.Raw(dangerousCVESQL, scannerUUID).QueryRows(&cves)
 	return cves, err
 }
+
+func countSQL(strSQL string) string {
+	return fmt.Sprintf(`select count(1) cnt from (%v) as t`, strSQL)
+}
+
+func (d *dao) CountVulnerabilities(ctx context.Context, registrationUUID string, projectID int64, tuneCount bool, query *q.Query) (int64, error) {
+	o, err := orm.FromContext(ctx)
+	if err != nil {
+		return 0, err
+	}
+	sqlStr := vulnerabilitySQL
+	params := []interface{}{registrationUUID}
+	if err := checkQFilter(query, allowedFieldTypeMap); err != nil {
+		return 0, err
+	}
+	sqlStr, params = applyVulFilter(ctx, sqlStr, query, params)
+	if tuneCount {
+		exceedLimit, err := d.countExceedLimit(ctx, sqlStr, params)
+		if err != nil {
+			return 0, err
+		}
+		log.Infof("exceedLimit: %v", exceedLimit)
```

Comment:
I think `Warnning` is better, and log should be clear like `The count is exceed to limit %v due to the tuneCount is enabled`.